duckduckgo/iOS
# BSK update for autofill never prompt to save for site feature

Task/Issue URL: https://app.asana.com/0/1201645688934642/1205190244970075/f
Tech Design URL:
CC:

**Description**:
BSK update for macOS support of never prompt to save for site feature



**Steps to test this PR**:
1. Just some light smoke testing steps -> Visit https://fill.dev/form/login-simple and login using new credentials. In the Save Login prompt tap `Never Ask for This Site`
2. Go back to https://fill.dev/form/login-simple
3. Login again with new credentials and confirm that you no longer get a prompt to save the login




---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
